app/models/price_policy.rb

```diff
@@ -32,6 +32,17 @@ def self.current
     current_for_date(Time.zone.now)
   end
 
+  def self.current_and_newest
+    # TODO: Fix bug that causes this to be neccessary (in truncate_existing_policies)
```

Comment:
I think expanding on what the bug is in this comment will be helpful when we stumble back on this in the future. Something like `Fix bug that allows overlapping price policies`.